configmap/README.md

```diff
@@ -0,0 +1,44 @@
+# Configmap
+
+Author: [Nick Santos](https://github.com/nicks)
+
+Helper functions for creating Kubernetes configmaps.
+
+## Functions
+
+### configmap_yaml
+
+```
+configmap_yaml(name: str, namespace: str = "", from_file: Union[str, List] = None, watch: bool = True): Blob
+```
+
+Returns YAML for a config map generated from a file.
+
+* `from_file` ( str ) – equivalent to `kubectl create configmap --from-file`
+* `watch` ( bool ) - auto-reload if the files change
+
+### configmap_create
+
+```
+configmap_create(name: str, namespace: str = "", from_file: Union[str, List] = None, watch: bool = True)
+```
+
+Deploys a config map. Equivalent to
+
+```
+load('ext://configmap', 'configmap_yaml')
```

Comment:
maybe I'm being nitty - feels slightly weird to have this in the block and say "equivalent to" when these functions don't have the side effects of `load`, and seems like it'd be just as clear without the `load`?

Reply:
done